__tests__/app.test.js

```diff
@@ -70,3 +70,39 @@ describe("/api/reviews", () => {
       });
   });
 });
+describe("/api/reviews/:review_id", () => {
+  test("GET - 200: responds with a review object", () => {
+    return request(app)
+      .get("/api/reviews/1")
+      .expect(200)
+      .then((res) => {
+        expect(res.body.review).toEqual({
+          review_id: expect.any(Number),
+          title: expect.any(String),
+          review_body: expect.any(String),
+          owner: expect.any(String),
```

Comment:
We want to be explicitly expecting the values associated with the given review here as we know what to expect -> instead this test should check that expected keys have correct values related to the requested review_id, e.g. NOT review_id: expect.any(Number)